MeetMatch.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { StyleSheet, Text, View, Image } from 'react-native';
+import { Font } from 'expo';
+
+export default class App extends React.Component {
+  state = {
+    fontLoaded: false,
+  };
+  async componentDidMount() {
+    await Font.loadAsync({
+      'double-bubble': require('./assets/fonts/Double_Bubble_shadow.otf'),
+      'bubble-body': require('./assets/fonts/Bubbleboddy-FatTrial.ttf'),
+      'source-sans': require('./assets/fonts/source-sans-pro.semibold.ttf')
+    });
+    this.setState({ fontLoaded: true }) ;
+  }
+  render(){
+    return(
+      <View style = { styles.container }>
+        <Text style = { styles.meetMatchText }>
+          MEET YOUR MATCH!
+        </Text>
+
```

Comment:
Remove this extra space

Reply:
I think this has been changed in the update-buttons branch. Do you mean the extra space on 23 or do you want "Meet your match" on line 20?